packages/storybook/stories/SendBox/snippets/RichTextSendBoxOnPasteCallback.snippet.tsx

```diff
@@ -0,0 +1,28 @@
+import { RichTextSendBox, FluentThemeProvider } from '@azure/communication-react';
+import React from 'react';
+
+const removeImageTags = (event: { content: DocumentFragment }): void => {
+  event.content.querySelectorAll('img').forEach((image) => {
+    // If the image is the only child of its parent, remove all the parents of this img element.
+    let parentNode: HTMLElement | null = image.parentElement;
+    let currentNode: HTMLElement = image;
+    while (parentNode?.childNodes.length === 1) {
+      currentNode = parentNode;
+      parentNode = parentNode.parentElement;
+    }
+    currentNode?.remove();
+  });
+};
+
+export const RichTextSendBoxOnPasteCallbackExample: () => JSX.Element = () => (
+  <FluentThemeProvider>
+    <div style={{ width: '31.25rem' }}>
+      <RichTextSendBox
+        onSendMessage={async () => {
+          return;
+        }}
+        onPaste={removeImageTags}
```

Comment:
The code snippet for the onPaste for both MessageThread and RTE Sendbox doesn't work when pasting plain text in.

Reply:
Could you clarify what do you mean and share steps to reproduce?

Reply:
Never mind. It was my MacOS not handling the copy paste properly.